src/anomalib/data/base/depth.py

```diff
@@ -35,30 +38,36 @@ def __getitem__(self, index: int) -> dict[str, str | torch.Tensor]:
         Returns:
             dict[str, str | torch.Tensor]: Dictionary containing the image, depth image and mask.
         """
-        image_path = self._samples.iloc[index].image_path
-        mask_path = self._samples.iloc[index].mask_path
-        label_index = self._samples.iloc[index].label_index
-        depth_path = self._samples.iloc[index].depth_path
+        image_path = self.samples.iloc[index].image_path
+        mask_path = self.samples.iloc[index].mask_path
+        label_index = self.samples.iloc[index].label_index
+        depth_path = self.samples.iloc[index].depth_path
 
-        image = read_image(image_path)
-        depth_image = read_depth_image(depth_path)
+        image = to_tensor(Image.open(image_path))
```

Comment:
can we use `from torchvision.io import read_image` for this to create tensor directly

Reply:
Unfortunately this does not work for all datasets. `torchvision.io` only supports `.png` and `.jpg` images, and some of our datasets have different image extensions (e.g. `.bmp` or `.tiff`).

Reply:
ah, yeah, would have been nice